frontend/src/components/SelectAsync/index.jsx

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect, useRef } from 'react';
 import { request } from '@/request';
 import useFetch from '@/hooks/useFetch';
 import { Select } from 'antd';
+import { useHistory } from 'react-router-dom/cjs/react-router-dom.min';
 
 export default function SelectAsync({
   entity,
```

Comment:
add param 'withRedirect' default is false ,
add param 'urlToRedirect' default is '/' ,

Reply:
done